scipy/integrate/tests/test_quadrature.py

```diff
@@ -397,3 +397,240 @@ def func(x):
         res = qmc_quad(func, 0, 1)
         ref = stats.norm.cdf(1, scale=2) - stats.norm.cdf(0, scale=2)
         assert_allclose(res.integral, ref, 1e-2)
+
+
+class TestCumulativeSimpson:
+    @pytest.mark.parametrize(
+        "y_func,x,dx,initial,int_y_func",
```

Comment:
```suggestion
        "y_func, x, dx, initial, int_y_func",
```
This is easier to read. Or even
```suggestion
        ("y_func", "x", "dx", "initial", "int_y_func"),
```
Same on different other occasions.

Reply:
Agreed @lorentzenchr , I have updated the parameters per your second suggestion.